garrysmod/gamemodes/terrortown/gamemode/cl_init.lua

```diff
@@ -367,7 +369,10 @@ function CheckIdle()
          RunConsoleCommand("say", "(AUTOMATED MESSAGE) I have been moved to the Spectator team because I was idle/AFK.")
 
          timer.Simple(0.3, function()
-                              RunConsoleCommand("ttt_spectator_mode", 1)
+                              RunConsoleCommand("ttt_spectator_mode", 1) -- If they want to bypass it now they still can but this way they still get fspecced

```

Comment:
The comment on this line only makes sense in context of this PR, so you should either explain more on what the concommand + net message are trying to do, or just remove it entirely.